docs/howto/kubernetes/index.md

```diff
@@ -0,0 +1,30 @@
+# Kubernetes in Cleura
+
+<!-- Config interpolation for the company name didn't work for the title here. -->
+
+In {{config.extra.company}} you can run containers in various ways, in every way you like. In {{config.extra.gui}} we offer UI for the
+Kubernetes management
+system Gardener and native OpenStack COE (Container orchestration engine) called Magnum.
+
+### Gardener
+
+Gardener is a Kubernetes-native system that implements the automated management and operation of Kubernetes clusters as a service. It
+provides a fully validated extensibility framework that can be adjusted to any programmatic cloud or infrastructure provider.
```

Comment:
This copies almost verbatim from <https://github.com/gardener/gardener#readme>. Please don't do that. Particularly if it contains marketing fluff like "Gardener is 100% Kubernetes-native" (what does that even mean)?